src/basic/log.h

```diff
@@ -31,6 +31,16 @@
 
 #include "macro.h"
 
+typedef enum LogRealm {
+        LOG_REALM_SYSTEMD,
+        LOG_REALM_UDEV,
+        _LOG_REALM_MAX,
+} LogRealm;
```

Comment:
hmm, can't we overload the priority field for this? The priority and facility bit definitions are public API, and explicitly define a mask of 0x3FF as LOG_PRIMASK|LOG_FACMASK, which pretty much leaves all bits >= 0x400 for our own, internal usage – as long as we strip them off before passing on.

With that in place we'd not have to change any of the internal logging APIs...